projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -42,7 +42,106 @@ const variations = compact( [
 				},
 			],
 		],
-		attributes: {},
+
+		attributes: {
+			variationName: 'default',
+		},
+	},
+	{
+		name: 'multistep-form',
+		title: __( 'Multistep Form', 'jetpack-forms' ),
+		description: __( 'Create a form that spans multiple steps.', 'jetpack-forms' ),
+		icon: {
+			foreground: getIconColor(),
+			src: renderMaterialIcon(
+				<Path d="M7 13h14V11H7v2zm0 4h14V15H7v2zm0-8h14V7H7v2zM4 11v2h2v-2H4zm0 4v2h2v-2H4zm0-8v2h2V7H4z" />
+			),
+		},
+		innerBlocks: [
+			[
+				'jetpack/form-progress-indicator',
+				{
+					labels: [
+						__( 'Contact Info', 'jetpack-forms' ),
+						__( 'Details', 'jetpack-forms' ),
+						__( 'Preferences', 'jetpack-forms' ),
+					],
+					activeStep: 0,
+					showLabels: true,
+				},
+			],
+			[
+				'jetpack/step-container',
+				{},
+				[
+					[
+						'jetpack/form-step',
+						{ title: __( 'Contact Information', 'jetpack-forms' ) },
+						[
+							[
+								'jetpack/field-name',
+								{ required: true, label: __( 'Full Name', 'jetpack-forms' ) },
+							],
```

Comment:
If we wanted to show off a bit more complicated form, we could split the field in two:

<img width="699" alt="image" src="https://github.com/user-attachments/assets/c45fef41-7298-4a4a-b3a6-9b71e1a67e3e" />


```suggestion
							[
								'jetpack/field-name',
								{ required: true, label: __( 'First name', 'jetpack-forms' ), width: 50 },
							],
							[
								'jetpack/field-name',
								{ required: true, label: __( 'Last name', 'jetpack-forms' ), width: 50 },
							],
```